python_scripts/linear_models_regularization.py

```diff
@@ -333,6 +357,14 @@
 best_alphas = [est[-1].alpha_ for est in cv_results["estimator"]]
 best_alphas
 
+# %% [markdown]
+# The optimal regularization strength is not unique. In fact, there is a set of
+# models leading to equivalent generalization performances.
```

Comment:
```suggestion
# The optimal regularization strength is not necessarily the same on all
# cross-validation iterations. But since we expect each cross-validation
# resampling to stem from the same data distribution, it is common practice
# to use the average value of the best `alpha` found on different
# cross-validation folds as our final estimate for the tuned `alpha`.
```